### ISSUE TITLE
Serializer dynamic include example from docs doesn't work

### ISSUE THREAD

**Issue by ctholho** [miragejs/miragejs#469]
I believe the docs are out of sync with reality. I tried the [example](https://miragejs.com/api/classes/serializer/#include) from the docs where you can define the serializer include as a function

```
Serializer.extend({
  include: function(request) {
    if (request.queryParams.posts) {
      return ['blogPosts'];
    } else {
      return [];
    }
  }
});
```

This didn't work. But, the simple example (`include: ['blogPosts']`) did work. So I figured maybe the if-condition is not right. Looking at `request.queryParams` I didn't find 'posts' but `{include: 'blogPosts'}. Guess, I should have expected that.

When I changed my code to this, it worked as expected.
```
Serializer.extend({
  include: function(request) {
    if (request.queryParams.include === blogPosts') {
      return ['blogPosts'];
    } else {
      return [];
    }
  }
});
```

**Comment by samselikoff** [comment]
Yes, you're right! I will update it to include the actual URL being requested as well.
